src/components/project/Featured/index.js

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import slug from 'slug';
```

Comment:
Let's not import `slug` to the project. 
Instead, create a slug data transformation inside gatsby-node.js graphql data call